### DISCUSSION TITLE
【求助】有人会吗？

### DISCUSSION THREAD

**Discussion by NOVAQUQ** [share121/inter-knot#232] [其他]
![Image_1723213838700](https://github.com/user-attachments/assets/a841e139-d51e-4425-94ca-3cd8744b8f5a)
如题（此处双关）
我帮一个朋友问的

**Comment by abeihua** [comment]
这委托我接不了！

**Comment by AL-Pinecore** [comment]
别再导啦！！！

**Comment by JingLu170** [comment]
v我时光机，换以前的我来就可以帮你

**Comment by chenger2024** [comment]
这个委托我接不了！

**Comment by sekiroissekiko** [comment]
这个委托我接不了，你问问别的绳匠

**Comment by KevinCoslina** [comment]
这个委托我接了

**Comment by KevinCoslina** [comment]
![Uploading IMG_4880.jpeg…]()

**Reply by NOVAQUQ** [reply]
图片没有上传成功啊大佬😂

**Comment by en-461** [comment]
另寻高明吧
